### DISCUSSION TITLE
API GetProduct add Attribute

### DISCUSSION THREAD

**Discussion by JanikHalder** [atrocore/atrocore#533] [Q&A] [answered]
Hi together,

is there a way to get the values of a specific product through the API?

If I call /Product/{id} the productAttributeValues aren't contained. 

Best,
Janik.

**Comment by rratsun** [comment, marked as answer]
Hello,

You can get product attribute value via ProductAttributeValue entity:
```/api/v1/ProductAttributeValue?maxSize=200&offset=0&sortBy=id&asc=true&where[0][type]=equals&where[0][attribute]=productId&where[0][value]=62da8aee3a237f4d```

or via Product entity relation:
```/api/v1/Product/62da8aee3a237f4de/productAttributeValues```
